lib/scala/library-manager/src/main/scala/org/enso/librarymanager/published/CachedLibraryProvider.scala

```diff
@@ -1,27 +1,24 @@
 package org.enso.librarymanager.published
 
 import nl.gn0s1s.bump.SemVer
-import org.enso.editions.{Editions, LibraryName}
-import org.enso.librarymanager.LibraryResolutionError
+import org.enso.editions.LibraryName
 import org.enso.librarymanager.published.cache.ReadOnlyLibraryCache
+import org.enso.librarymanager.resolved.LibraryRoot
 
-import java.nio.file.Path
 import scala.annotation.tailrec
-import scala.util.Try
 
 /** A [[PublishedLibraryProvider]] that just provides libraries which are
   * already available in the cache.
   */
 class CachedLibraryProvider(caches: List[ReadOnlyLibraryCache])
-    extends PublishedLibraryProvider
-    with PublishedLibraryCache {
+    extends PublishedLibraryCache {
```

Comment:
This comment is not valid anymore - this class no longer implements the `PublishedLibraryProvider` interface, like suggested in the doc comment.